code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -711,3 +711,59 @@
 	item_state = "syndie_helm"
 	item_color = "syndi"
 	armor = list("melee" = 40, "bullet" = 50, "laser" = 30, "energy" = 15, "bomb" = 35, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+
+/obj/item/clothing/head/helmet/space/hardsuit/deathsquad
+	name = "deathsquad helmet"
+	desc = "That's not red paint. That's real blood."
+	icon_state = "hardsuit0-deathsquad"
+	item_state = "deathsquad"
+	item_color = "deathsquad"
+	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	vision_flags = SEE_MOBS
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE //don't render darkness while wearing these
+	see_in_dark = 8
+	HUDType = MEDHUD
+	strip_delay = 130
+
+	sprite_sheets = list()
+	sprite_sheets_obj = list()
+
+/obj/item/clothing/suit/space/hardsuit/deathsquad
+	name = "deathsquad suit"
+	desc = "A heavily armored, advanced space suit that protects against most forms of damage."
+	icon_state = "deathsquad"
+	item_state = "swat_suit"
+	allowed = list(/obj/item/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/restraints/handcuffs,/obj/item/tank/internals,/obj/item/kitchen/knife/combat)
+	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	strip_delay = 130
+	dog_fashion = /datum/dog_fashion/back/deathsquad
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/deathsquad
+	jetpack = /obj/item/tank/jetpack/suit
+
+	sprite_sheets = list()
+	sprite_sheets_obj = list()
+
+/obj/item/clothing/head/helmet/space/hardsuit/deathsquad/beret
+	name = "officer beret"
+	desc = "An armored beret commonly used by special operations officers."
+	icon_state = "beret_officer"
+	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```

Comment:
```suggestion
```
It inherits this from the regular DS helmet, Shouldn't need to re-state it here.